pkg/disk/controllerserver.go

```diff
@@ -224,6 +226,15 @@ func (cs *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 
 	diskType, diskID, diskPL, err := createDisk(req.GetName(), snapshotID, requestGB, diskVol, req.Parameters[TenantUserUID])
 	if err != nil {
+		var aliErr *alicloudErr.ServerError
+		if errors.As(err, &aliErr) {
```

Comment:
it will panic because aliErr is nil?

Reply:
`&aliErr` is of type `**alicloudErr.ServerError` and `&aliErr` is not nil. Besides, this is already covered by the csi-sanity test cases.